maps/DRO.json

```diff
@@ -0,0 +1,251 @@
+                  "http://sdr.sul.stanford.edu/models/sdr3-page.jsonld",
+                  "http://sdr.sul.stanford.edu/models/sdr3-photograph.jsonld",
+                  "http://sdr.sul.stanford.edu/models/sdr3-manuscript.jsonld",
+                  "http://sdr.sul.stanford.edu/models/sdr3-map.jsonld",
+                  "http://sdr.sul.stanford.edu/models/sdr3-media.jsonld",
+                  "http://sdr.sul.stanford.edu/models/sdr3-track.jsonld",
+                  "http://sdr.sul.stanford.edu/models/sdr3-webarchive-binary.jsonld",
+                  "http://sdr.sul.stanford.edu/models/sdr3-webarchive-seed.jsonld" ]
+      },
+      "citation": {
+        "description": "Citation for the resource, including identifier, label, version, and a persistent URL to the object with SDR at the very least.",
+        "type": "string"
+      },
+      "depositor": {
+        "description": "The Agent (User, Group, Application, Department, other) that deposited the DRO into SDR.",
+        "type": "object",
+        "properties": {
+          "required": ["name"],
+          "name": {
+            "type": "string"
+          },
+          "sunetID": {
+            "type": "string"
+          }
+        }
+      },
+      "is_target_of": {
+        "description": "An Annotation instance that applies to the DRO.",
+        "type": "string",
+        "format": "URI"
+      },
+      "label": {
+        "description": "Primary processing label (can be same as title) for a DRO.",
+        "type": "string"
+      },
+      "version": {
+        "description": "Version for the DRO within SDR.",
+        "type": "integer"
+      },
+      "administrative": {
+        "type": "object",
+        "description": "",
+        "required": ["created", "isDescribedBy", "preserve"],
+        "properties": {
+          "created": {
+            "description": "When the resource in SDR was created.",
+            "type": "dateTime"
+          },
+          "deleted": {
+            "description": "",
+            "type": "Boolean"
+          },
+          "gravestoneMessage": {
+            "description": "",
+            "type": "string",
+          },
+          "isDescribedBy": {
+            "description": "PURL/XML file (represented by S3 IRI?) that is a traditional representation of the metadata for the DRO.",
+            "type": "string",
+            "format": "URL"
+          },
+          "lastUpdated": {
+            "description": "When the resource in SDR was last updated.",
+            "type": "dateTime"
+          },
+          "partOfProject": {
+            "description": "",
+            "type": "string"
+          },
+          "preserve": {
+            "description": "Whether or not the DRO should be preserved.",
+            "type": "boolean",
+          },
+          "remediatedBy": {
+            "description": "The Agent (User, Group, Application, Department, other) that remediated a DRO in SDR.",
+            "type": "object",
```

Comment:
Could this be an array? Could it have undergone multiple remediations?

Reply:
WIP.

Reply:
Yes, I saw WIP. Can we collaborate on this? How/when would you like comments and questions?

Reply:
no longer relevant. comment made before completion of first pass.